WalletWasabi/JsonConverters/NetworkJsonConverterNg.cs

```diff
@@ -29,6 +29,11 @@ public class NetworkJsonConverterNg : JsonConverter<Network>
 
 		networkString = reader.GetString()!.Trim();
 
+		if (networkString == nameof(Network.TestNet))
+		{
+			networkString = nameof(Network.RegTest);
```

Comment:
what is this?

Reply:
Since we are using Wasabi data, it can happen that someone starts the client on TestNet. This converts it to RegTest.
I decided next to RegTest as that is our testing network.